CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Export C++ definitions in CMake config file
 - Fix Doxyfile python bindings directory ([#110](https://github.com/Simple-Robotics/aligator/pull/110))
 - Fix for eigenpy 3.3 ([#121](https://github.com/Simple-Robotics/aligator/pull/121))
+- Fix finite difference function in python unittest
```

Comment:
You must move this in *Unreleased* section. Also add the PR link.

Reply:
Done.